backend/testfiles/httpclient/v0/todo/request-query-param-string-emoji-key.test

```diff
@@ -16,7 +16,7 @@ Content-Length: LENGTH
 
 [test]
 (let query = { "😭👨‍👩‍👧‍👦" = "emoji" }
- let response = (PACKAGE.Darklang.Stdlib.HttpClient.request "get" "http://URL" query HttpClient.jsonContentType_v0) |> Builtin.unwrap
+ let response = (PACKAGE.Darklang.Stdlib.HttpClient.request "get" "http://URL" query [HttpClient.ContentType.json_v0]) |> Builtin.unwrap
```

Comment:
Should we use the FQName here, or is it unnecessary?`PACKAGE.Darklang.Stdlib.HttpClient.ContentType.json_v0]`
instead of 
`HttpClient.ContentType.json_v0]`